docs/api/other-api/adapter.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "@react-router/{adapter}"
+---
+
+# Server Adapters
+
+## Official Adapters
+
+Idiomatic React Router apps can generally be deployed anywhere because React Router adapts the server's request/response to the [Web Fetch API][web-fetch-api]. It does this through adapters. We maintain a few adapters:
+
+- `@react-router/architect`
+- `@react-router/cloudflare`
+- `@react-router/express`
+
+These adapters are imported into your server's entry and are not used inside your React Router app itself.
+
+If you initialized your app with `npx create-react-router@latest` with something other than the built-in [React Router App Server][rr-serve] (`@react-router/serve`), you will note a `server/index.js` file that imports and uses one of these adapters.
+
+<docs-info>If you're using the built-in React Router App Server, you don't interact with this API</docs-info>
+
+Each adapter has the same API. In the future we may have helpers specific to the platform you're deploying to.
+
+## `@react-router/express`
+
+[Reference Documentation ↗](https://api.reactrouter.com/v7/modules/_react_router_express.html)
+
+Here's a simple example with Express:
+
+```tsx
+import express from "express";
+
+const BUILD_PATH = "./build/server/index.js";
+const PORT = Number.parseInt(process.env.PORT || "3000");
+
+const app = express();
+
+app.use(
+  "/assets",
+  express.static("build/client/assets", {
+    immutable: true,
+    maxAge: "1y",
+  }),
+);
+app.use(express.static("build/client", { maxAge: "1h" }));
+app.use(await import(BUILD_PATH).then((mod) => mod.app));
+
+app.listen(PORT, () => {
+  console.log(
+    `Server is running on http://localhost:${PORT}`,
+  );
+});
+```
+
+### Migrating from the React Router App Server
+
+If you started an app with the [React Router App Server][rr-serve] but find that you want to take control over the Express server and customize it, it should be fairly straightforward to migrate way from `@react-router/serve`.
```

Comment:
```suggestion
You can refer to the [Express template][express-template] as a reference, but here are the main changes you should have to make:
```

Reply:
I'm confused lol - this sentence is below so this suggestion would duplicate it.  Do you just want to eliminate the "If you started an app with..." sentence?